packages/cloud_asset_inventory/data_stream/asset_inventory/manifest.yml

```
@@ -15,115 +15,157 @@ streams:
     template_path: aws.yml.hbs
     enabled: false
     vars:
-      - name: access_key_id
+      - name: aws.account_type
+        title: Account type
+        multi: false
+        required: true
+        show_user: true
+        type: select
+        options:
+          - text: Single Account
+            value: single-account
+        default: single-account
+      - name: aws.credentials.type
+        title: Credentials type
+        multi: false
+        required: true
+        show_user: true
+        type: select
+        options:
+          - text: CloudFormation
+            value: cloud_formation
+          - text: Assume Role (manual)
+            value: assume_role
+          - text: Direct Access Keys (manual)
+            value: direct_access_keys
+          - text: Temporary Keys (manual)
+            value: temporary_keys
+          - text: Shared Credentials (manual)
+            value: shared_credentials
+      - name: aws.access_key_id
         type: text
         title: Access Key ID
         multi: false
         required: false
         show_user: true
         secret: false
-      - name: secret_access_key
+        description: Required when using Direct Access Keys or Temporary Keys
+      - name: aws.secret_access_key
         type: text
         title: Secret Access Key
         multi: false
         required: false
         show_user: true
         secret: true
-      - name: session_token
+        description: Required when using Direct Access Keys or Temporary Keys
+      - name: aws.session_token
         type: text
         title: Session Token
         multi: false
         required: false
         show_user: true
         secret: false
-      - name: shared_credential_file
+        description: Required when using Temporary Keys
+      - name: aws.shared_credential_file
         type: text
         title: Shared Credential File
         multi: false
         required: false
         show_user: false
-        description: Directory of the shared credentials file
-      - name: credential_profile_name
+        description: Directory of the shared credentials file; required when using Shared Credentials
```

Nit: just for consistency 
```suggestion
        description: Directory of the Shared Credentials file; required when using Shared Credentials
```